src/styles/blocks/card.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  width: 200px;
+  border: 1px solid #{$card-border-color};
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__image {
+    align-self: center;
+    width: 160px;
+    height: 134px;
+
+    display: block;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: #{$main-accent-color};
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+
+    color: #{$secondary-color};
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 24px;
+  }
+
+  &__review {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+
+    color: #{$main-accent-color};
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    &-text {
```

Comment:
better don't do it, it's hard to comprehend and price can be confused with a block